### PR TITLE
clean nonexistent metrics in e2e test

### PR DESCRIPTION
**Pull request by danielqsj**
**What type of PR is this?**

/kind cleanup
/sig test
/sig instrumentation

**What this PR does / why we need it**:

The following four metrics can not be found now:
```
kubelet_container_manager_latency_microseconds
kubelet_docker_errors
kubelet_generate_pod_status_latency_microseconds
kubelet_sync_pods_latency_microseconds
```

If they are really nonexistent, then we should remove them in e2e tests.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/framework/metrics_util.go [hunk 1]
```diff
@@ -184,14 +184,10 @@ var InterestingControllerManagerMetrics = []string{
 }
 
 var InterestingKubeletMetrics = []string{
-	"kubelet_container_manager_latency_microseconds",
-	"kubelet_docker_errors",
```
Comment (BenTheElder) [inline test/e2e/framework/metrics_util.go]: it looks like these are now in the dockershim to be clear https://github.com/kubernetes/kubernetes/commit/331628b7dcd637f3e5622b654977955ebb17ce5a#diff-bb5d427394b7c4081ea5dba3b3465b76

Comment (BenTheElder) [reply test/e2e/framework/metrics_util.go]: @yujuhong looks like you moved these, thoughts on this?

Comment (danielqsj) [reply test/e2e/framework/metrics_util.go]: Maybe replace this by `docker_operations_errors` is good one .

Comment (danielqsj) [reply test/e2e/framework/metrics_util.go]: /cc @brancz

Comment (brancz) [reply test/e2e/framework/metrics_util.go]: That sounds good to me :+1:

Comment (danielqsj) [reply test/e2e/framework/metrics_util.go]: @brancz @BenTheElder :  added the new metrics `kubelet_docker_operations_errors_total` to replace the deprecated/missing metrics `kubelet_docker_errors` . PTAL

**Comment by brancz** [comment]
cc @deads2k

**Comment by BenTheElder** [comment]
/unassign
I would like a testing approver from SIG-Node to handle this

**Comment by danielqsj** [comment]
PR rebased

/assign @liggitt @vishh for sig-node approve

**Comment by brancz** [comment]
cc @derekwaynecarr

**Comment by danielqsj** [comment]
friendly ping @derekwaynecarr @liggitt : could you please help review this ? thanks

**Pull request merged**